proto/api.proto

```diff
@@ -412,6 +423,39 @@ service Api {
     };
   }
 
+  rpc GetOrCreateUser(GetOrCreateUserRequest) returns (GetOrCreateUserResponse) {
+    option (google.api.http) = {
+      post: "/api/v1/drift/user"
```

Comment:
I'm ok with this endpoint being `/drift` since it seems like it could be Drift specifically and not applicable to future perpetual markets we might want to support –– can you confirm that you expect this to be the case? For the two below this though they should be part of the `/trade/perp` categroy and have the appropriate tags